qa/acceptance/spec/shared_examples/updated.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../spec_helper'
+require          'logstash/version'
+
+RSpec.shared_examples "updated" do |logstash|
+
+  before (:all) { logstash.snapshot }
+  after  (:all) { logstash.restore }
+
+  it "can update on #{logstash.host}" do
+    logstash.install("5.0.0-alpha2", "./")
```

Comment:
I agree, I guess we should somehow make it available for most of the parts here, this is also need in the provision scripts, will see if I think of a better way to do it.